.github/workflows/main.yaml

```
@@ -0,0 +1,13 @@
+name: Build and deploy runner image to k8s cluster
+
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * 0'  # every week at midnight on sunday
+
+jobs:
+  build:
+    uses: nuxeo/nuxeo-web-ui-gh-runner/.github/workflows/build.yaml@master
+  deploy:
+    uses: nuxeo/nuxeo-web-ui-gh-runner/.github/workflows/deploy.yaml@master
```

Shouldn't this depends on the build?
```suggestion
  deploy:
    needs: build
    uses: nuxeo/nuxeo-web-ui-gh-runner/.github/workflows/deploy.yaml@master
```

---

Ups 😱  nice catch 🤦  Not being able to run this to test is tricky.